.github/workflows/vercel-deployment.yml

```diff
@@ -0,0 +1,20 @@
+name: Vercel Preview Deployment
+
+on:
+  push:
+
+jobs:
+  vercel-deployment-dev:
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-vercel-deployment.yml@issue-33-create-vercel-deployment-workflow
```

Comment:
I believe this PR was merged as the changes are on main:

https://github.com/ai-cfia/github-workflows/blob/main/.github/workflows/workflow-vercel-deployment.yml

Reply:
it wasn't merged I just copied the file from main to test the development